scripts/build/cross

```diff
@@ -1,12 +1,18 @@
 #!/usr/bin/env bash
+#
+# Build a binary for all supported platforms
+#
 
 set -eu -o pipefail
 
 BUILDDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 
-echo "Building all binaries"
+echo "Building binaries for all platforms"
 SHELL=/bin/bash parallel ::: \
-    "$BUILDDIR/linux-cross" \
     "$BUILDDIR/windows" \
     "$BUILDDIR/osx" \
+    "GOARCH=amd64   $BUILDDIR/binary" \
```

Comment:
I guess right now it assumes they it's always run in a linux container, but that's not necessarily the case. 

I've added `GOOS=linux`